### PR TITLE
F/apply command

### PR DESCRIPTION
**Pull request by pranavkulkarni**

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -42,6 +48,13 @@ function rollbackOpts (yargs) {
     .choices('env', ['dev', 'qa', 'prod'])
 }
 
+function applyOpts (yargs) {
+  return yargs
+    .describe('file-path', 'relative path of a file to be run on a cluster')
```
Comment (dmfenton) [inline bin/cli.js]: can we make this one `-f` or `--file` like kubectl?

File: load-resources.js [hunk 2]
```diff
@@ -35,6 +34,9 @@ async function loadResources (root, values = {}, options = {}) {
 }
 
 function buildManifest (root, options) {
+  if (options.rootIsSpecificFile) {
+    return [path.join(process.cwd(), root)]
```
Comment (dmfenton) [inline load-resources.js]: you should be able to detect this by using the `path` module

**Review by dmfenton — CHANGES_REQUESTED** [review]

**Comment by pranavkulkarni** [comment]
I used `fs` module instead of path to check if it is a file

File: bin/cli.js [hunk 3]
```diff
@@ -42,6 +48,13 @@ function rollbackOpts (yargs) {
     .choices('env', ['dev', 'qa', 'prod'])
 }
 
+function applyOpts (yargs) {
+  return yargs
+    .describe('file', 'relative path of a file to be run on a cluster')
```
Comment (dmfenton) [inline bin/cli.js:53]: can you also do alias('file', f) please?

File: test/load-resources.spec.js [hunk 4]
```diff
@@ -19,3 +19,14 @@ test('load a list', async function (t) {
   t.equal(ClusterRole.kind, 'ClusterRole', 'loaded second type correctly')
   t.equal(ClusterRoleBinding.kind, 'ClusterRoleBinding', 'loaded first type correctly')
 })
+
+test('load a specific file', async function (t) {
+  t.plan(3)
+  const resources = await loadResources('test/fixtures/namespaces/test/cluster-autoscaler.hbs', { minSize: 1, maxSize: 10, env: 'dev-east-1', clusterDomain: 'foo.bar.com' }, { rootIsSpecificFile: true })
```
Comment (dmfenton) [inline test/load-resources.spec.js]: remove the rootIsSpecific file from this test

File: bin/cli.js (hunk 3 above)
Comment (pranavkulkarni) [reply bin/cli.js:53]: oh, that's neat, just added

**Review by dmfenton — APPROVED** [review]

**Pull request merged**